cmd/template_cmd.go

```diff
@@ -105,7 +111,11 @@ func TemplateCreate(c *cli.Context) error {
 		templateIn["run_list"] = utils.RemoveDuplicates(strings.Split(c.String("run-list"), ","))
 	}
 	if c.IsSet("cookbook-versions") {
-		templateIn["cookbook_versions"] = (*params)["cookbook-versions"]
+		cbIn, err := convertFlagParamsToCookbookVersions(c, c.String("cookbook-versions"))
+		if err != nil {
+			formatter.PrintFatal("Cannot parse input cookbook-versions parameters", err)
+		}
+		templateIn["cookbook_versions"] = cbIn
 	}
 	if c.IsSet("configuration-attributes") {
```

Comment:
Shouldn't we work with the `--configuration-attributes-from-file` flag here as well?

Reply:
yes! i thought it was done!

Reply:
Done